src/main/resources/portal.properties.EXAMPLE

```diff
@@ -8,7 +8,10 @@ db.host=localhost
 db.portal_db_name=cbioportal
 db.driver=com.mysql.jdbc.Driver
 db.connection_string=jdbc:mysql://localhost/
-db.tomcat_resource_name=jdbc/cbioportal
+# set tomcat_resource_name when using dbconnector=jndi instead of the defaault
+# dbconnector=dbcp
```

Comment:
@inodb, did you test this? I was under the impression that the `dbconnector` setting only worked if it was supplied as a `-D` property to the JVM, since it's being used as a Spring applicationContext.xml profile in the same way as the other `authenticate` setting listed in  https://github.com/cBioPortal/cbioportal/issues/6109

Reply:
Never mind, I see you're already tracking this as a followup task in https://github.com/cBioPortal/cbioportal/issues/6148